src/components/PageHeader/PageHeader.tsx

```diff
@@ -1,26 +1,26 @@
-import React from "react";
-import Banner from "../Banner";
-import ImageWithStickers from "../ImageWithStickers";
+// import React from "react";
```

Comment:
Do you just want to delete all of PageHeader? I think this will cause an error in the Page Header story if it doesn't know what PageHeader is